draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -331,27 +333,27 @@
                 The trigger resource has a state attribute.  
                 dCDN creates new triggers in "pending" state. Once dCDN starts processing a trigger, the trigger 
                 state is set to "active". 
-                Once the the trigger processing is complete, the state is set to either "complete" or "failed", 
+                Once the trigger processing is complete, the state is set to either "complete" or "failed", 
                 depending on the processing outcome.
         </t>
         <t>
                 uCDN MAY request dCDN to cancel a trigger. If such request is accepted, the trigger state is changed to 
                 "cancelling", and when the cancellation is complete, the trigger state changes to "cancelled".
         </t>        
         <t>         
-                For full description of the Trigger resource, please refer to <xref target="ci-trigger-resource"/>.
+                For full description of the trigger resource, please refer to <xref target="ci-trigger-resource"/>.
         </t>
 
    </section>
    <section title="Trigger Collection" anchor="model-for-cdni-triggers-trigger-collection-resource">
         <t>
-        Trigger Collection is a resource that represents all trigger resources, belonging to particular uCDN.
-        dCDN maintains one Trigger Collection resource for each uCDN. 
+        Trigger collection is a resource that represents all trigger resources, belonging to a particular uCDN.
```

Comment:
Nit: Since we're not treating this as a proper noun, i.e. "Trigger Collection", perhaps it would be more appropriate to say "A trigger collection is a resource..."?